staging/src/k8s.io/apiserver/pkg/admission/plugin/policy/validating/metrics/metrics_test.go

```diff
@@ -33,6 +33,9 @@ func TestNoUtils(t *testing.T) {
 	metrics := []string{
 		"apiserver_validating_admission_policy_check_total",
 		"apiserver_validating_admission_policy_check_duration_seconds",
+		"apiserver_validating_admission_policy_active_policies",
+		"apiserver_validating_admission_policy_active_bindings",
```

Comment:
Do we need two of them? We don't care the active policy number hence when one policy be binded twice, I guess user would wanna count as 2 instead of 1 so using active binding number as policy count makes sense to me.

Reply:
My expectation when developing this was for the policy count to be 1 and the binding count to be 2.